browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2419,11 +2419,14 @@ Are you sure you want to do this?
       <ph name="YEARLY_AMOUNT">%1$s</ph> / yr.
     </message>
     <message name="IDS_BRAVE_VPN_RENEWAL_TEXT" desc="Brave VPN plan text">
-      Try free for 7 days. After 7 days, you will be charged the plan price. All subscriptions are auto-renewed but can be cancelled before renewal.
+      Try free for 6 days. After 7 days, you will be charged the plan price. All subscriptions are auto-renewed but can be cancelled before renewal.
```

Comment:
This change is quite weird. First sentence doesn't reconcile with the rest of the text.

Reply:
i agree. i had the same question when i was updating. but it was confirmed in figma spec.

Reply:
@samartnik i have updated the days here.